autotest/build.sbt

```
@@ -69,9 +69,18 @@ installOptions := {
 def optPath(bc: Config, p: String) = if (bc.hasPath(p)) Some(file(bc.getString(p))) else None
 
 autotestInstallerZip := {
-  val bc = autotestBuildConfig.value
-  optPath(bc, "install.zip").orElse(optPath(bc, "install.dir").map(d =>
-    (d * "equella-installer-*.zip").get.head))
+  val bc                     = autotestBuildConfig.value
+  val equellaFullVersion     = equellaVersion.value
+  val installerFileName      = s"equella-installer-${equellaFullVersion.semanticVersion}.zip"
+  val installerDirectory     = (target in LocalProject("Installer")).value
+  val installerAbsoloutePath = installerDirectory / installerFileName
```

Don't forget to fix the typo @SammyIsConfused mentioned:

```suggestion
  val installerAbsolutePath = installerDirectory / installerFileName
```